mimic_2_causal_inference.ipynb

```diff
@@ -0,0 +1,1044 @@
+{
```

Comment:
<p>-&gt; This graph can now be fed into</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>